control-plane/agents/src/common/errors.rs

```diff
@@ -394,6 +394,8 @@ pub enum SvcError {
     },
     #[snafu(display("Invalid property name '{property_name}' for the volume '{id}'"))]
     InvalidSetProperty { property_name: String, id: String },
+    #[snafu(display("Cannot set '{attribute}' on Replica '{replica}''s Lvol xattr"))]
+    ReplicaSetXattrFailed { attribute: String, replica: String },
```

Comment:
Let's not use XAttr, that's dataplane speak, control-plane does need to know, also we might have other backends soon..

```suggestion
    #[snafu(display("Cannot set entity_id '{entity_id}' on Replica '{replica}'"))]
    ReplicaSetEntityFailed { entity: String, replica: String },
```

Reply:
How about ReplicaSetPropertyFailed { attribute: String, replica: String } ? We could reuse this if some other property set fails?

Reply:
Sure, that seems good too